test/unit/spread.spec.ts

```diff
@@ -87,3 +87,22 @@ describe("in object literal", () => {
         `.expectToMatchJsResult();
     });
 });
+
+const spreadCases = [
+    "1, 2, ...[3, 4, 5]",
+    "...[1, 2], 3, 4, 5",
+    "1, 2, ...'spread', 4, 5",
+    "1, [2], ...[3, 4, 5]",
+    "...[1, 2, 3], 4, ...[5, 6]",
+];
```

Comment:
It would be nice to add a test case with a nested array value, to test that `.concat` doesn't flatten args